test/e2e/apimachinery/webhook.go

```diff
@@ -755,6 +756,36 @@ func testWebhook(f *framework.Framework) {
 	Expect(err).NotTo(HaveOccurred(), "failed to create configmap %s in namespace: %s", configmap.Name, skippedNamespaceName)
 }
 
+func testBlockingConfigmapDeletion(f *framework.Framework) {
+	By("create a configmap that should be denied by the webhook")
```

Comment:
the text reads ambiguous to me. The creation should not be denied but the deletion should be